src/reference-designs/tko-on-vsphere-with-tanzu-nsxt.md

```diff
@@ -0,0 +1,446 @@
+
+After a Supervisor Cluster is created, the vSphere administrator creates vSphere Namespaces. When initially created, the namespace has unlimited resources within the Supervisor Cluster. The vSphere administrator defines the limits for CPU, memory, storage, as well as the number of Kubernetes objects that can run within the namespace. These limits are configured per vSphere Namespace.
+
+![vSphere Namespace](img/tko-on-vsphere-with-tanzu-nsxt/image10.png)
+
+To provide tenants access to namespaces, the vSphere administrator assigns permission to users or groups available within an identity source that is associated with vCenter Single Sign-On.
+
+Once the permissions are assigned, the tenants can then access the namespace to create Tanzu Kubernetes Clusters using the YAML file and Cluster API.
+
+## Supported Component Matrix
+
+The following table provides the component versions and interoperability matrix supported with the reference design:
+
+| Software Components      | Version                   |
+| ------------------------ | ------------------------- |
+| Tanzu Kubernetes Release | 1.21.2                    |
+| VMware vSphere ESXi      | ESXi - 7.0 U3\* and later |
+| VMware vCenter (VCSA)    | 7.0 U3 and later          |
+| VMware NSX-T             | 3.1 and later             |
+
+## vSphere with Tanzu Storage
+
+vSphere with Tanzu integrates with shared datastores available in the vSphere infrastructure. The following types of shared datastores are supported:
+
+- vSAN
+- VMFS
+- NFS
+- vVols
+
+vSphere with Tanzu uses storage policies to integrate with shared datastores. The policies represent datastores and manage the storage placement of such objects as control plane VMs, container images, and persistent storage volumes.
+
+Before you enable vSphere with Tanzu, create storage policies. The storage polices are used by the Supervisor Cluster and namespaces. Depending on your vSphere storage environment, you can create several storage policies to represent different classes of storage.
+
+vSphere with Tanzu is agnostic about which option you choose. [VMware vSAN](https://docs.vmware.com/en/VMware-vSAN/index.html) is a recommended storage solution for both Supervisor Cluster and workload cluster VMs.
+
+For Kubernetes stateful workloads, vSphere with Tanzu installs the [vSphere Container Storage interface (vSphere CSI)](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/index.html) to automatically provision Kubernetes persistent volumes for pods.
+
+While the default vSAN storage policy can be used, administrators should evaluate the needs of their applications and configure a specific [vSphere Storage Policy](https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.storage.doc/GUID-89091D59-D844-46B2-94C2-35A3961D23E7.html). vSAN storage policies describe classes of storage, such as SSD and NVME, as well as quotas for your clusters.
```

Comment:
[The comma before such as is used correctly](https://www.grammarly.com/blog/such-as-comma/#:~:text=How%20to%20Use%20Such%20As,very%20end%20of%20the%20sentence.)

'as well as' is needed because and is already used to differentiate between SSD and NVME. Changed to 'as well as quota definitions for the VSAN cluster.', as it makes more clear the purpose of the policy.